examples/gazebo_iris/iris_with_standoffs_demo/model.config

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
```

Comment:
I guess we could drop this folder?

Reply:
No. Both the folders `iris_with_standoffs` and `iris_with_standoffs_demo` are being used. I could try to consolidate them into one model directory.

Reply:
Might make sense, unless they're really for different things. There is also a stray .tar.gz file in there.

Reply:
Removed the tar.gz. Will shift them into one folder and commit

Reply:
:+1: